astropy/wcs/utils.py

```diff
@@ -632,3 +632,193 @@ def pixel_to_skycoord(xp, yp, wcs, origin=0, mode='all', cls=None):
     coords = cls(frame.realize_frame(data))
 
     return coords
+def _linear_transformation_fit(params, lon, lat, x, y, w_obj):	
```

Comment:
Two blank spaces before a function.
```suggestion


def _linear_transformation_fit(params, lon, lat, x, y, w_obj):	
```